EventGenerator/test/corsikaTest.fcl

```diff
@@ -26,6 +26,7 @@ source: {
     @table::targetParams
     seed: 1
     resample: false
+    compact: false
```

Comment:
Is the default then not to use the compact format?

Reply:
I left the full version as default because I wanted to check the differences with a large dataset in the compact format. I did that and I can't see any difference, switching to compact as default.